airbyte-integrations/base-java/.dockerignore

```diff
@@ -0,0 +1,4 @@
+*
+!Dockerfile
+!build
+!javabase.sh
```

Comment:
where is this file?

Reply:
gone.

Reply:
this actually came back since i changed the implementation. i think your question is answered now though.